docker/docker-local.yml

```
@@ -2,17 +2,23 @@ services:
   mdv_app:
     build:
       context: .
-      dockerfile: Dockerfile
+      dockerfile: Dockerfile-local 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)